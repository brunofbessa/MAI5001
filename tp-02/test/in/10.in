1
3
1.123,2.345,3.456,4.567,5.678,X
1.321,2.543,3.654,4.765,5.111,5.876,X
1.111,2.222,3.333,4.444,5.555,6.666,7.777,X