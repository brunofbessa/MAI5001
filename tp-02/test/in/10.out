1.111,1.123,1.321,2.222,2.345,2.543,3.333,3.456,3.654,4.444,4.567,4.765,5.111,5.555,5.678,5.876,6.666,7.777